doc/tutorials.dox

```diff
@@ -49,6 +49,7 @@ Using YARP devices:
 \li \subpage yarp_config_files
 
 More software tutorials:
+\li \subpage yarp_pointcloud
```

Comment:
I would add  this page also on the  Interoperability and advanced use section in the main page of YARP.

Reply:
Is it possible to do it without restarting all the tests ? :grimacing:

Reply:
@Nicogene let's be flexible and add this change after the merge 😉 (if you like to have the green lights within the PR)

Reply:
https://github.com/robotology/yarp/pull/1597/files#diff-64c78214510ead1279a75f3949512cc3 
done